packages/component/src/Styles/StyleSet/CustomerSatisfactoryAttachment.ts

```diff
@@ -0,0 +1,150 @@
+const DARK_THEME_SELECTOR = '@media (forced-colors: none) and (prefers-color-scheme: dark)';
+const FORCED_COLORS_SELECTOR = '@media (forced-colors: active)';
+const LIGHT_THEME_SELECTOR = '@media (forced-colors: none) and (prefers-color-scheme: light)';
+const NOT_FORCED_COLORS_SELECTOR = '@media (forced-colors: none)';
+
+const DISABLED_SELECTOR = '&:disabled, &[aria-disabled="true"]';
+const NOT_DISABLED_SELECTOR = '&:not(:disabled):not([aria-disabled="true"])';
+
+export default function CustomerSatisfactoryAttachment() {
+  return {
+    '&.webchat__customer-satisfactory': {
+      alignItems: 'flex-start',
+      display: 'flex',
+      flexDirection: 'column',
+      fontFamily: 'var(--webchat__font--primary)',
+      gap: 8,
+      padding: '10px 12px'
+    },
+
+    '& .webchat__customer-satisfactory__radio-group': {
+      display: 'flex',
+      flexDirection: 'column',
+      gap: 8
+    },
+
+    '& .webchat__customer-satisfactory__label': {
+      margin: 0
+    },
+
+    '& .webchat__customer-satisfactory__star-bar': {
+      display: 'flex',
+
+      [FORCED_COLORS_SELECTOR]: {
+        color: 'ButtonBorder'
+      },
+
+      [NOT_FORCED_COLORS_SELECTOR]: {
+        [DARK_THEME_SELECTOR]: {
+          color: '#White'
+        },
+
+        [LIGHT_THEME_SELECTOR]: {
+          color: '#242424'
+        }
+      }
+    },
+
+    '& .webchat__customer-satisfactory__star-button': {
+      backgroundColor: 'transparent',
+      color: 'unset',
```

Comment:
alpha :)

Reply:
Actually, it don't need to be `unset`. I was playing between `<input type="radio">` and `<button role="radio">`.

Wonder what do you mean by "alpha"?